### DISCUSSION TITLE
Supabase email Notification

### DISCUSSION THREAD

**Discussion by drjoeycadieux** [supabase/supabase#21860] [Questions] [answered]
Hello, 

Is it possible to receive notification or try to create that from supabase so when data come in i receive and send a notification email. Thanks

**Comment by GaryAustin1** [comment, marked as answer]
There is nothing built in, but it can be done.
You could use webhooks to call an edge function which calls an email provider API.
You could use a trigger function and the http or pg_net extensions to call an edge function or email provider API.
You could use cron to check for changes and "bundle up" several updates in a single email by calling an edge function or the email provider API.

**Reply by drjoeycadieux** [reply]
Ok Thanks
